appveyor.yml

```diff
@@ -16,9 +16,11 @@ before_build:
   - ps: .\build\Set-BuildVersion -Version $env:APPVEYOR_BUILD_VERSION -Suffix $env:VERSION_SUFFIX
   - nuget sources add -name Exceptionless -source https://www.myget.org/F/exceptionless/api/v3/index.json
   - appveyor-retry dotnet restore -v Minimal
+  - appveyor-retry dotnet restore .\src\Foundatio.Redis.Signed -v Minimal
```

Comment:
I don't think these dot net restores were left over from when we had two project folders?